ads/ads.extern.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Long this of, uhm, stuff the ads code needs to compile.
```

Comment:
Will fix comment.

Using an extern is the right way to do this. It is literally what they are for. It is fragile and individual networks could contribute their official file, but I don't care. Their problem :)